config/config.go

```diff
@@ -18,6 +18,7 @@ var ErrEmptyEnv = errors.New("env is needed, but empty")
 // Config is configuration for simulator.
 type Config struct {
 	Port                int
+	APIURL              string
```

Comment:
nits: It is unclear whether this name indicates the URL of the simulator server or the URL of the kube-apiserver.
So, please rename it to `KubeAPIServerURL` or something like that.